crates/quickjs-wasm-rs/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Make `JSContextRef::wrap_rust_value` private. Similar to
```

Comment:
Should this be considered removed? The function is still there, but from the library consumer's point of view, it's a feature that's been removed from the public API. I don't feel strongly about it though.

Reply:
I consider it, but for someone reading the code, or even comparing differences between both versions, I concluded that this is probably the best wording to capture what happened.